pkg/kritis/metadata/metadata.go

```diff
@@ -24,17 +24,17 @@ import (
 
 type Fetcher interface {
 	// Vulnerabilities returns package vulnerabilities for a given image.
-	Vulnerabilities(containerImage string) ([]Vulnerability, error)
+	Vulnerabilities(containerImage string, auths []kritisv1beta1.AttestationAuthority) ([]Vulnerability, error)
 	// Create Attesatation Occurrence for an image.
 	CreateAttestationOccurence(note *grafeasv1beta1.Note,
-		containerImage string,
-		pgpSigningKey *secrets.PGPSigningSecret) (*grafeasv1beta1.Occurrence, error)
-	//AttestationNote getches a Attestation note for an Attestation Authority.
+		containerImage string, pgpSigningKey *secrets.PGPSigningSecret,
+		aa *kritisv1beta1.AttestationAuthority) (*grafeasv1beta1.Occurrence, error)
+	//AttestationNote fetches an Attestation note for an Attestation Authority.
 	AttestationNote(aa *kritisv1beta1.AttestationAuthority) (*grafeasv1beta1.Note, error)
 	// Create Attestation Note for an Attestation Authority.
 	CreateAttestationNote(aa *kritisv1beta1.AttestationAuthority) (*grafeasv1beta1.Note, error)
 	//Attestations get Attestation Occurrences for given image.
-	Attestations(containerImage string) ([]PGPAttestation, error)
+	Attestations(containerImage string, auths []kritisv1beta1.AttestationAuthority) ([]PGPAttestation, error)
```

Comment:
Could you change this to a single auth, instead of an array?  A comment in review.go provides some more justification for this.  Additionally, errors are somewhat more meaningful for a single auth.